packages/edit-site/src/components/editor-canvas-container/style.scss

```diff
@@ -1,5 +1,4 @@
 .edit-site-editor-canvas-container {
-	background: $white; // Fallback color, overridden by JavaScript.
```

Comment:
I've removed this rule so that there isn't a tiny bit of white bleeding in at the edge of the Style Book when viewed in browse mode. This was most noticeable on Chrome when hovering over the main frame when the background color of the site was dark. There'd be a thin sliver of the background color of the canvas container poking through. This removal fixes that.